shot-android/src/main/java/com/karumi/shot/ScreenshotTest.kt

```diff
@@ -131,9 +133,26 @@ interface ScreenshotTest {
     private fun disableAnimatedComponents(view: View) {
         runOnUi {
             hideEditTextCursors(view)
+            hideScrollViewBars(view)
         }
     }
 
+    private fun hideScrollViewBars(view: View) {
+        view.childrenViews<ScrollView>().forEach {
+            hideViewBars(it)
+        }
+
+        view.childrenViews<HorizontalScrollView>().forEach {
+            hideViewBars(it)
+        }
```

Comment:
I think this should cover as well all of `NestedScrollView` childrenViews

Reply:
Hi @fmontesino, I think we can add this in a different PR.